schematic/utils/general.py

```diff
@@ -224,6 +224,9 @@ def create_temp_folder(path: str) -> str:
         path(str): a directory path where all the temporary files will live
     Returns: returns the absolute pathname of the new directory.
     """
+    if not os.path.exists(path):
```

Comment:
Is there any reason we would want an error to be thrown if the path doesn't exist?

Reply:
Not in this case I don't think